api/config/service/authn/v1/authn.proto

```diff
@@ -18,6 +18,7 @@ message OIDC {
   // The subject is mapped from the JWT token's email claim by default.
   // Set this field to the JWT token's claim name to override the subject.
   string subject_claim_name_override = 6;
+  string groups_claim_name_override = 7;
```

Comment:
Please add a comment documenting the default behavior and what the override does (like the above comment).